lib/fathead/mdn_css/parse.pl

```diff
@@ -431,6 +439,22 @@ sub parse_fragment_data {
     }
 }
 
+sub parse_initial_value {
+    my ($table_properties) = @_;
+
+    #first <tr> Contains the initial value
+    my $tr = $table_properties->find('tr')->first;
+    my $th = $tr->at('th');
+    my $initial_value;
+    if ($th) {
+        my $a = $th->at('a');
+        if ( $a && $a->text =~ /Initial value/ ) {
+            $initial_value = $tr->all_text;
+        }
+    }
+    return $initial_value;
```

Comment:
Let's put the Initial value in it's own `<p>` to separate it out. I think the markup should be:

```html
<p><b>Initial Value: </b><em>$initial_value</em></p>
```